extensions/amp-fx-flying-carpet/0.1/amp-fx-flying-carpet.css

```diff
@@ -45,9 +45,16 @@ amp-fx-flying-carpet > .-amp-fx-flying-carpet-clip > .-amp-fx-flying-carpet-cont
   position: fixed !important;
   top: 0 !important;
   width: 100%;
+  height: 100%;
   /*
    * Included to force slow webkit browsers (Android) into rendering the
    * container using the GPU.
    */
   -webkit-transform: translateZ(0) !important;
+
+  /* Center contents in the container */
+  display: flex;
```

Comment:
I think it will play well. But I'm still testing around. Just added a good improvement to support responsive layout as well. PTAL.